src/app/public/src/modules/blockquote/blockquote.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="stache-blockquote">
+  <div class="stache-blockquote-icon">
+    <i class="fa fa-quote-left" aria-hidden="true"></i>
```

Comment:
Some accessibility? Nice! We should add a task to do an accessibility assessment for our stuff. Maybe Matt Gregg could help us with that.